integration/js/script/run-test

```diff
@@ -123,6 +133,10 @@ case $1 in
       r meeting_leave_video_test.config.json
       r reconnection_test.config.json
       ;;
+   video_test_app)
+      r test_app_video_test.config.json
+      r test_app_video_pin_test.config.json
```

Comment:
We eventually want to use the test app for the video E2E test too

Reply:
it didn't work when i added it yesterday so need to check that later.